packages/bpk-docs/src/pages/NeoHomePage/HomePage.js

```diff
@@ -78,6 +78,7 @@ const CARD_CONTENTS = [
   //   href: ROUTES.STYLE_GUIDE,
   //   image: StyleGuideImage,
   //   icon: StyleGuideIcon,
+  //   iconWidth: '120px',
```

Comment:
`px`? 😱

Reply:
I was following the convention set on line 90 of this file where we set iconWidth for the GitHub icon to 153px.

Having said that, Will's going to provide a new icon so this will probably go away anyway.

Reply:
We should use `rem`'s for everything.